<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="todayEventsLabel">
      <property name="text">
       <string notr="true">[Сегодня](123)</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_9">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="eventList"/>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignRight|Qt::AlignVCenter">
              <widget class="QLabel" name="enrollByReferLbl">
               <property name="text">
                <string>[Стать участником по коду](123)</string>
               </property>
               <property name="textFormat">
                <enum>Qt::MarkdownText</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_10">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="newEventBtn">
               <property name="text">
                <string>Создать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteEventBtn">
               <property name="text">
                <string>Отменить</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="eventCtrlFrame">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
              <widget class="QLabel" name="eventNameLbl">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Название: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eventName">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
              <widget class="QLabel" name="eventDTLbl">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Дата и время:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="eventTime"/>
             </item>
             <item>
              <widget class="QDateEdit" name="eventDate">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="locale">
                <locale language="Russian" country="Russia"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
              <widget class="QLabel" name="viewReferLbl">
               <property name="text">
                <string>Код приглашения</string>
               </property>
               <property name="textFormat">
                <enum>Qt::MarkdownText</enum>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight|Qt::AlignVCenter">
              <widget class="QLabel" name="viewParticipantsLbl">
               <property name="text">
                <string>Участники</string>
               </property>
               <property name="textFormat">
                <enum>Qt::MarkdownText</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
